docs/src/content/docs/es/guides/components.mdx

```diff
@@ -22,9 +22,7 @@ import OtroComponente from '../../componentes/OtroComponente.astro';
 
 <UnComponente prop="algo" />
 
-<OtroComponente>
-  Los componentes también pueden contener **contenido anidado**.
-</OtroComponente>
+<OtroComponente>Los componentes también pueden contener **contenido anidado**.</OtroComponente>
```

Comment:
Ahhh, the joys of auto-formatting content. We often use a shorter line length in code sample code than in regular code so it stays legible on narrower screens. Is it possible to use a shorter line length in `.md` and `.mdx`?

Example of this change overflowing on the left and current version on the right:
<img width="1179" alt="Screenshot comparison" src="https://github.com/withastro/starlight/assets/357379/7bafb043-4779-4b48-aa0b-b4fe372b73dd">